Project/Assets/Prairie/Framework/Script/Twine/NodeInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class NodeInfo : MonoBehaviour {
```

Comment:
I think that this class does more than just hold Node Info since it now has the `TriggerNode` method. Could we rename it `TwineNode` or something like that? To me, a `NodeInfo` class sounds like something that just holds info in some fields (like this class used to do)